addon/components/hyper-table-v2/column.ts

```diff
@@ -24,13 +31,35 @@ export default class HyperTableV2Column extends Component<HyperTableV2ColumnArgs
       .finally(() => {
         this.loadingHeaderComponent = false;
       });
+
+    if (args.column.definition.orderable || args.column.definition.filterable) {
+      args.handler.renderingResolver
+        .lookupFilteringComponent(args.column.definition)
+        .then((resolution) => {
+          this.filteringComponent = resolution;
+        })
+        .finally(() => {
+          this.loadingFilteringComponent = false;
```

Comment:
This should really be in the finally step? What happens if an error happen ?

Reply:
we want to stop the loading whatever happens. there is no error handling for now unfortunately.